keras/api/_tf_keras/keras/utils/bounding_boxes/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""DO NOT EDIT.
+
+This file was autogenerated. Do not edit it by hand,
+since your modifications would be overwritten.
+"""
+
+from keras.src.layers.preprocessing.image_preprocessing.bounding_boxes.converters import (
+    affine,
+)
+from keras.src.layers.preprocessing.image_preprocessing.bounding_boxes.converters import (
+    clip_to_images,
```

Comment:
I wonder if this would be more usable via `clip_to_image_size(boxes, image_size)`, e.g. `clip_to_size(boxes, images.shape[1:3])`? You may not always have image tensors on hand, so it's nice to be able to do `clip_to_size(boxes, (12, 34))`.

Reply:
Yes, name seems misleading it actually takes height and width not the image tensors. I will change the name to `clip_to_image_size` which is apt actually.